docs/content/get-started/set-up.md

```diff
@@ -67,6 +65,10 @@ Once you have navigated to the **Workloads** tab:
 
 Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [**Unity Support** plugin](https://github.com/JetBrains/resharper-unity) for a better experience.
 
+**Step 5.** Restart your machine.
+
+> If you do not restart your machine, you may experience errors when opening a GDK project.
+
```

Comment:
Text is tip-top.

Looking at the formatting of the page, we are gonna lose this step under the VS and Rider bits of additional info, especially in the Windows instructions  - I think we need to either:
(1) Move the info about VS and Rider to the right as a Note (like the notes above) - not sure that would work for Windows as loads of text and an image.
(2) Move the info about VS and Rider to below step 5, with a note on step 4 to look at the info below.
(3) Something else.